quicklogic/qlf_k4n8/utils/repacker/tests/lut_padding/lut1.eblif

```diff
@@ -0,0 +1,6 @@
+.model top
```

Comment:
Is this test folder part of github actions run? We need to see that these tests are run every time whenever a PR is created.

Reply:
Yes it is. I've connected the repacker tests to the `all_quicklogic_tests` target. See the changes to `quicklogic/qlf_k4n8/utils/CMakeLists.txt`.

Also when you browse logs from CI runs you should encounter lines like:
```
repacker/tests/lut_padding/test_lut_padding.py::test_lut_padding[1-1]
```
which indicated that the tests were run.